modules/ROOT/pages/using-policies.adoc

```diff
@@ -36,7 +36,7 @@ For details about the configuration parameters for each of the policies, see the
 
 == See Also
 
```

Comment:
If these are cross-refs to Task topics, the topic titles should be changed to imperative: Review, Share, Request.

Reply:
Done! Thanks!